tests/instruments/picoquant/test_library_wrapper.py

```diff
@@ -1,17 +1,15 @@
 import unittest.mock
```

Comment:
why did we have to remove some tests?

Reply:
License reasons. I will put the ones we can test later on, when I have the necessary licensing text in.